packages/connectivity/connectivity/pubspec.yaml

```diff
@@ -20,7 +20,7 @@ dependencies:
     sdk: flutter
   meta: "^1.0.5"
   connectivity_platform_interface: ^1.0.2
-  connectivity_macos: ^0.0.3
+  connectivity_macos: ^0.1.0
```

Comment:
Remember to publish connectivity_macos to pub.dev, or this dep will break checks.